resources/js/features/achievements/components/CreateAchievementTicketMainRoot/CreateAchievementTicketForm/IssueSelectField.tsx

```diff
@@ -0,0 +1,114 @@
+import { useLaravelReactI18n } from 'laravel-react-i18n';
+import type { FC } from 'react';
+import { useFormContext } from 'react-hook-form';
+
+import {
+  BaseFormControl,
+  BaseFormDescription,
+  BaseFormField,
+  BaseFormItem,
+  BaseFormLabel,
+} from '@/common/components/+vendor/BaseForm';
+import {
+  BaseSelect,
+  BaseSelectContent,
+  BaseSelectItem,
+  BaseSelectTrigger,
+  BaseSelectValue,
+} from '@/common/components/+vendor/BaseSelect';
+import { Trans } from '@/common/components/Trans';
+
+import type { CreateAchievementTicketFormValues } from './useCreateAchievementTicketForm';
+
+export const IssueSelectField: FC = () => {
+  const { t } = useLaravelReactI18n();
+
+  const form = useFormContext<CreateAchievementTicketFormValues>();
+
+  return (
+    <BaseFormField
+      control={form.control}
+      name="issue"
+      render={({ field }) => (
+        <BaseFormItem className="flex w-full flex-col gap-1 sm:flex-row sm:items-center">
+          <BaseFormLabel
+            htmlFor="issue-select"
+            className="text-menu-link sm:mt-[13px] sm:min-w-36 sm:self-start"
+          >
+            {t('Issue')}
+          </BaseFormLabel>
+
+          <div className="flex w-full flex-col gap-1">
+            <BaseFormControl>
+              <BaseSelect onValueChange={field.onChange} defaultValue={field.value ?? undefined}>
+                <BaseSelectTrigger id="issue-select" className="sm:w-full md:w-96">
+                  <BaseSelectValue placeholder={t('Select an issue')} />
+                </BaseSelectTrigger>
+
+                <BaseSelectContent>
+                  <BaseSelectItem value="DidNotTrigger">{t('Did not trigger')}</BaseSelectItem>
+                  <BaseSelectItem value="TriggeredAtWrongTime">
+                    {t('Triggered at the wrong time')}
+                  </BaseSelectItem>
+                  <BaseSelectItem value="NetworkIssue">
+                    {t('Triggered, but not showing as earned on site')}
+                  </BaseSelectItem>
+                </BaseSelectContent>
+              </BaseSelect>
+            </BaseFormControl>
+
+            {field.value ? (
+              <BaseFormDescription className="text-menu-link light:text-neutral-800">
```

Comment:
Should these example texts be muted like the helper text for the other fields?

Reply:
Yes, I think so. The most important helper text is the highlighted text near the description field. Otherwise, they should probably all be muted. Updated in latest.

Note that I did not use the actual muted text color for this -- the contrast is not high enough against the page's background.